hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/retry/RetryPolicies.java

```diff
@@ -639,19 +647,24 @@ public FailoverOnNetworkExceptionRetry(RetryPolicy fallbackPolicy,
     
     public FailoverOnNetworkExceptionRetry(RetryPolicy fallbackPolicy,
         int maxFailovers, int maxRetries, long delayMillis, long maxDelayBase) {
+      this(fallbackPolicy, maxFailovers, maxRetries, delayMillis, maxDelayBase, 2);
+    }
+    public FailoverOnNetworkExceptionRetry(RetryPolicy fallbackPolicy,
+        int maxFailovers, int maxRetries, long delayMillis, long maxDelayBase, int nnSize) {
       this.fallbackPolicy = fallbackPolicy;
       this.maxFailovers = maxFailovers;
       this.maxRetries = maxRetries;
       this.delayMillis = delayMillis;
       this.maxDelayBase = maxDelayBase;
+      this.nnSize = nnSize;
     }
 
     /**
      * @return 0 if this is our first failover/retry (i.e., retry immediately),
```

Comment:
The comments here looks need to be updated too.

Reply:
@cndaimin Thanks for the review. I will add the comments later.

Reply:
@cndaimin I updated the code. Please take a look.